src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/Images.cs

```diff
@@ -3,35 +3,47 @@
 
 using System;
 using System.Windows;
+using System.Windows.Media;
 using System.Windows.Media.Imaging;
 
 namespace NuGet.PackageManagement.UI
 {
     internal static class Images
     {
-        public static readonly BitmapImage DefaultPackageIcon;
+        public static readonly BitmapSource DefaultPackageIcon;
 
         static Images()
         {
             // in VS, look up the icon via pack://application url
             if (Application.Current != null)
             {
-                DefaultPackageIcon = new BitmapImage();
-                DefaultPackageIcon.BeginInit();
+                var image = new BitmapImage();
+                image.BeginInit();
 
                 // If the DLL name changes, this URI would need to change to match.
-                DefaultPackageIcon.UriSource = new Uri("pack://application:,,,/NuGet.PackageManagement.UI;component/Resources/packageicon.png");
+                image.UriSource = new Uri("pack://application:,,,/NuGet.PackageManagement.UI;component/Resources/packageicon.png");
 
                 // Instead of scaling larger images and keeping larger image in memory, this makes it so we scale it down, and throw away the bigger image.
                 // Only need to set this on one dimension, to preserve aspect ratio
-                DefaultPackageIcon.DecodePixelWidth = 32;
+                image.DecodePixelWidth = 32;
```

Comment:
Let's keep the default pixel width definition in one place.

```suggestion
                image.DecodePixelWidth = PackageItemListViewModel.DecodePixelWidth;
```